src/Symfony/Component/AssetMapper/ImportMap/Resolver/JsDelivrEsmResolver.php

```diff
@@ -205,6 +218,47 @@ public function downloadPackages(array $importMapEntries, callable $progressCall
             throw new RuntimeException(sprintf('Error %d downloading packages from jsDelivr for "%s". Check your package names. Response: ', $response->getStatusCode(), $packages).$response->getContent(false), 0, $e);
         }
 
+        $extraFileErrors = [];
+        download_extra_files:
+        $packageFileResponses = $extraFileResponses;
+        $extraFileResponses = [];
+        foreach ($packageFileResponses as $package => $responses) {
+            foreach ($responses as [$response, $extraFile, $packageName, $version]) {
+                if (200 !== $response->getStatusCode()) {
+                    $extraFileErrors[] = [$package, $response];
+                    continue;
+                }
+
+                $extraFiles = [];
+
+                $content = $response->getContent();
+                if (str_ends_with($extraFile, '.css')) {
+                    $content = $this->makeImportsBare($content, $dependencies, $extraFiles, ImportMapType::CSS, $extraFile);
+                }
+                $contents[$package]['extraFiles'][$extraFile] = $content;
+
+                if (0 !== \count($extraFiles)) {
+                    $extraFileResponses[$package] = [];
+                    foreach ($extraFiles as $newExtraFile) {
+                        $extraFileResponses[$package][] = [$this->httpClient->request('GET', sprintf(self::URL_PATTERN_DIST_CSS, $packageName, $version, $newExtraFile)), $newExtraFile, $packageName, $version];
+                    }
+                }
+            }
+        }
+
+        if ($extraFileResponses) {
+            goto download_extra_files;
+        }
```

Comment:
This got a bit complex because it needs to be recursive (`first.css` has a `url()` pointing to `second.css`, so then we need to check `second.css` also for `url()`). However, the recursion is tested, though I don't know of any examples "in the wild" to try this on.

I imagine vendor packages importing one css file from another would be quite rare because CSS files are always packaged into one to reduce requests (even with HTTP2 this is still important for CSS files, as they are page-render-blocking and you can't discover that you need to download `second.css` until you download `first.css`).